testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/settings/AppSettings.kt

```diff
@@ -180,6 +192,18 @@ private class Settings(context: Context) {
                 null
         set(isRequired) = prefs.edit().putBoolean(REQUIRE_SIGNATURE_VERIFICATION, isRequired!!).apply()
 
+    var isProdVersionEnabled: Boolean?
+        get() =
+            if (prefs.contains(IS_PROD_VERSION_ENABLED))
+                prefs.getBoolean(IS_PROD_VERSION_ENABLED, false)
+            else
+                null
+        set(isRequired) = prefs.edit().putBoolean(IS_PROD_VERSION_ENABLED, isRequired!!).apply()
+
+    var baseUrl: String?
+        get() = prefs.getString(BASE_URL, null)
+        set(baseUrl) = prefs.edit().putString(BASE_URL, baseUrl).apply()
+
     var projectId: String?
```

Comment:
Don't we also need to switch between the default Prod Project Id/Key and the Staging Id/Key? 

@laubengaier How are you handling this on iOS?

Reply:
@corycaywood @laubengaier i thought qa can use id/key by inputing them, if ios has done it automatically i will change too..